docs/reference/extensibility/NuGet-Cross-Platform-Plugins.md

```diff
@@ -70,7 +73,10 @@ CI/CD scenarios and power users can use environment variables to override the be
 
 - `NUGET_NETFX_PLUGIN_PATHS` - defines the plugins that will be used by the .NET Framework based tooling (NuGet.exe/MSBuild.exe/Visual Studio). Takes precedence over `NUGET_PLUGIN_PATHS`. (NuGet version 5.3+ only)
 - `NUGET_NETCORE_PLUGIN_PATHS` - defines the plugins that will be used by the .NET Core based tooling (dotnet.exe). Takes precedence over `NUGET_PLUGIN_PATHS`. (NuGet version 5.3+ only)
-- `NUGET_PLUGIN_PATHS` - defines the plugins that will be used for that NuGet process, priority preserved. If this environment variable is set, it overrides the convention based discovery. Ignored if either of the framework specific variables is specified.
+- `NUGET_PLUGIN_PATHS`

```

Comment:
This comment is about the `Plugin installation and discovery` section.

We should call ouit that search for nuget-plugin-* executables here as well. We only have it in the general plugin requirements right now.